client/state/timezones/reducer.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import {
+	DESERIALIZE,
+	SERIALIZE,
+
+	TIMEZONES_RECEIVE,
+	TIMEZONES_REQUEST,
+	TIMEZONES_REQUEST_FAILURE,
+	TIMEZONES_REQUEST_SUCCESS,
+} from 'state/action-types';
+import {
+	rawOffsetsSchema,
+	timezonesSchema
+} from './schema';
+
+const offsets = createReducer( [], {
+	[ TIMEZONES_RECEIVE ]: ( state, { rawOffsets = [] } ) => rawOffsets.slice( 0 )
+}, rawOffsetsSchema );
+
+const timezones = createReducer( {}, {
+	[ TIMEZONES_RECEIVE ]: ( state, { timezonesByContinent } ) => ( timezonesByContinent )
+}, timezonesSchema );
+
+export const items = combineReducers( {
```

Comment:
> Should we just flatten offsets and timezones into state.timezones ? If worried about state.timezones.timezones, maybe it's more appropriately named as byContinent anyways?

I think do you meant to `state.timezones.timezones_by_contienet`? I prefer to keep them separately since they are different things.`byContinent` sounds great, though.